README.md

```diff
@@ -1 +1,142 @@
-# angular-using-ngmessages-readme
\ No newline at end of file
+# Using ngMessages / ngMessage
+
+## Overview
+
+When our inputs have lots of validation rules, writing out condition `ng-if` statements for *every* type of validation can be very verbose. This is where a new module comes in - `ngMessages`!
```

Comment:
yes, that's great! We have a great sense of the problem right away from the previous README.